TrelloAutotest/Pages/TrelloCreateWorkSpacePage.cs

```diff
@@ -40,9 +44,16 @@ public void DeleteWorkSpace(string workSpaceName)
 
             driver.FindElement(WorkSpaceSelectors.WorkSpaceDeleteButton).Click();
             driver.FindElement(BaseSelectors.ConfirmButton).Click();
+
+            return this;
         }
 
-        public bool IsWorkSpaceDeleted(string workSpaceName) => driver.FindElements(WorkSpaceSelectors.DeletedWorkSpace)
-                .Any(x => x.Text.Contains(workSpaceName, StringComparison.OrdinalIgnoreCase));
+        public TrelloCreateWorkSpacePage IsWorkSpaceDeleted(string workSpaceName)
+        {
+            Assert.IsFalse(driver.FindElements(WorkSpaceSelectors.DeletedWorkSpace)
+                .Any(x => x.Text.Contains(workSpaceName, StringComparison.OrdinalIgnoreCase)));
```

Comment:
It is not a proper way to cut the line of code - it suggests that there is _Any_ after _IsFalse_. If the line is too long it is better to this e.g. in this way:
![image](https://user-images.githubusercontent.com/22776410/124507980-41646780-ddcf-11eb-8a66-ec11af044f7f.png)